translations/introduction.tex/es.po

```diff
@@ -37,6 +37,11 @@ msgid ""
 "several different Factions, led by their Heroes, battle for supremacy. "
 "Choose your Faction and Hero and banish your unruly enemies from these lands!"
 msgstr ""
+"\\hypertarget{Heroes of Might and Magic III}{\\textbf{Heroes of Might and "
+"Magic III: El juego de mesa}} es un juego de mesa RPG de estrategia táctica para 1-3 "
```

Comment:
```suggestion
"Magic III: The Board Game}} es un juego de mesa RPG de estrategia táctica para 1-3 "
```
I suggest leaving the name of the original. You have the translation right next to it anyway.

Reply:
I think the same, but I was not sure